astroquery/esa/hubble/tests/test_esa_hubble.py

```diff
@@ -86,19 +84,19 @@ def get_dummy_tap_handler(self):
         return dummyTapHandler
 
     def test_download_product_errors(self):
-        ehst = ESAHubbleClass(self.get_dummy_tap_handler())
+        ehst = ESAHubbleClass(tap_handler=self.get_dummy_tap_handler(), show_messages=False)
```

Comment:
Can we have one `show_messages=True` test, too (with mocking some status messages)?

Reply:
Test included.